build.gradle

```diff
@@ -88,6 +88,16 @@ task travisRelease {
     }
 }
 
+task installGitPreCommitHook(type: Copy) {
```

Comment:
- [ ] - Can you move those 2 tasks into "gradle/precommit.gradle" script and use "apply from" to load the functionality? This way, we encapsulate complexity in separate files and keep the code clean and easy to understand.